cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -6005,4 +5947,72 @@
     </description>
   </property>
 
+<!--  Artifact localizer settings -->
+  <property>
+    <name>artifact.localizer.preload.list</name>
+    <value></value>
+    <description>
+      A command separated list of system artifacts to be preloaded in
+      artifact-localizer
+    </description>
+  </property>
+
+  <property>
+    <name>artifact.localizer.preload.version.limit</name>
+    <value>2</value>
+    <description>
+      Count of latest versions to be preloaded in artifact-localizer
```

Comment:
nit: add comments that it may go OOM if it's set to high value and CDAP updates multiple times.